jsonformer_claude/main.py

```diff
@@ -174,47 +97,6 @@ async def prefix_matches(self, progress) -> bool:
         self.debug("[prefix_matches]", result)
         return result
 
-    async def generate_string(self) -> str:
-        prompt = self.get_prompt() + '"'
-        progress = self.get_progress() + '"'
```

Comment:
this forces a string to start with ", does that still happen with these changes?

Reply:
Ah interesting -- I am checking that the first character is a string.  I _think_ this is equivalent although I'm not updating progress to check for the character, it'll just return None and then be retried I think (gotta check exactly how the retry works)